docs/.doxybook/templates/breadcrumbs.tmpl

```diff
@@ -0,0 +1,19 @@
+{% if exists("moduleBreadcrumbs") -%}
```

Comment:
I don't know if this is mentioned anywhere, but this is a custom template meant to mimic cppreference as much as possible with doxybook2. The implications are:

- So, whenever we are in doubt, we can always check cppreference to make sure we have the expected output. 
- This involved a lot of workarounds, especially for function overloads, which means we have an opportunity for some other project here or an opportunity to contribute to doxybook2.

Reply:
That's interesting and seems pretty useful. I'll try to understand this a little bit deeper.